com.doycho.euterpe.gtk.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id" : "com.doycho.euterpe.gtk",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "3.38",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "euterpe-gtk",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "libhandy",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dprofiling=false",
+                "-Dintrospection=enabled",
+                "-Dgtk_doc=false",
+                "-Dtests=false",
+                "-Dexamples=false",
+                "-Dvapi=false",
+                "-Dglade_catalog=disabled"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/libhandy",
+                    "tag" : "1.2.1"
+                }
+            ]
+        },
+		"shared-modules/python-cryptography/python-cryptography.json",
+		"pypi-dependencies.json",
+        {
+            "name" : "euterpe-gtk",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/ironsmile/euterpe-gtk.git",
+					"tag": "v0.1.0",
+					"commit": "2333d22a60742b1d4a864cfab68c2f458325ce83"
+                }
+            ]
+        }
+    ],
+    "build-options" : {
+        "env" : {        }
+    }
```

Comment:
```suggestion
    ]
```

Reply:
Are you sure? This will make `build-options` a key into the `modules` array. I am pretty sure it must in the top level object.

Reply:
I'm pretty this doesn't do what you think it does.

*hint* it's empty, it does nothing, it is not needed and this is what the "suggestion" is about: removing it.

Reply:
Oh, I see. I misunderstood the suggestion diff. Anyway, removed it.